### DISCUSSION TITLE
service "crisp"

### DISCUSSION THREAD

**Discussion by yoan34** [AmauriC/tarteaucitron.js#905] [Ideas]
Bonjour, je propose également d'ajouter **crisp**, une solution pertinente de chatbot français.

**marqueur HTML**:
`<script type="text/javascript">window.$crisp=[];window.CRISP_WEBSITE_ID=LE_SUPER_ID;(function(){d=document;s=d.createElement("script");s.src="https://client.crisp.chat/l.js";s.async=1;d.getElementsByTagName("head")[0].appendChild(s);})();</script>`<br>
**Liste des cookies qui apparait après un test local**:
- crisp-client%2Fsession%2Fehub89b4-fcb7-49e0-8670-3b5146cc0569: "session_524c13a6-429f-4630-91e3-e0i6b6l9c6d2"
- crisp-client%2Fsocket%2Fehub89b4-fcb7-49e0-8670-3b5146cc0569:"1"<br>

**Politique de cookie**:
https://crisp.chat/fr/privacy/

NOTE: Le site stipule que le consentement de base n'est pas requis (peut-être pour ça que le service n'est pas présent):
https://help.crisp.chat/en/article/crisp-chatbox-cookie-ip-policy-1147xor/

**Comment by AmauriC** [comment]
C'est noté ; )
https://github.com/AmauriC/tarteaucitron.js/issues/906
